### PR TITLE
Add menu to card faces

### PR DESCRIPTION
**Pull request by derricklee43**
https://user-images.githubusercontent.com/39753553/187006796-866c41de-6c36-4eb3-a3bc-b3ca28cba65f.mov

## Summary of Changes
- Added menu for each Card Face
- Added ability to set card face language individually
- Added ability to swap card face content
- Updated white hover color
- Added helper component for language selectors

### CODE REVIEW

**Review by jiejasonliu — APPROVED** [review]
a suggestion, but from a UX perspective, I think it makes sense to close the dropdown after an action has been performed

(i.e. after swapping, it should hide the dropdown)

File: src/components/flashcard/card-face/card-face.tsx [hunk 1]
```diff
@@ -33,7 +39,15 @@ export const CardFace = ({
   return (
     <div className={`card-face ${className} ${variant}`} onClick={handleCardFaceClick}>
       {isReadonly && hasText && getSpeaker()}
-      {variant === 'active' && getKebabMenu()}
+      {variant === 'active' && (
+        <CardMenu
+          language={cardContent.language as any}
```
[the remark this answers is not in the document]
Comment (derricklee43) [reply src/components/flashcard/card-face/card-face.tsx]: > a suggestion, but from a UX perspective, I think it makes sense to close the dropdown after an action has been performed
> 
> (i.e. after swapping, it should hide the dropdown)

Haha, I was going back and forth between these two options earlier. I switched it to close after click

**Comment by derricklee43** [comment]
> a suggestion, but from a UX perspective, I think it makes sense to close the dropdown after an action has been performed
> 
> (i.e. after swapping, it should hide the dropdown)

Haha, I was going back and forth between these two options earlier. I switched it to close after click

File: src/components/drop-down-options/drop-down-options.scss [hunk 2]
```diff
@@ -2,14 +2,10 @@
...
 .c-drop-down-options {
-  @include box-shadow;
+  @include floating-card;
   @include hover-pointer;
   width: 100%;
   margin-top: 2px;
-  position: absolute;
-  z-index: 1;
-  background-color: $white;
-  border-radius: $standard-radius + 1px; //hides corners of div
```
[the remark this answers is not in the document]
Comment (derricklee43) [reply src/components/drop-down-options/drop-down-options.scss:12]: haha, true. Must have forgotten to put it back in. It's fixed now

**Pull request merged**